packages/core/src/utils.js

```diff
@@ -662,12 +662,37 @@ export function stubExistingAdditionalProperties(
   return schema;
 }
 
+const _resolveCondition = (schema, rootSchema, formdata) => {
```

Comment:
none of the private method has _ in the method name, maybe best to keep it without

Reply:
Done